Sources/Apodini/Semantic Model Builder/Model/Endpoint.swift

```diff
@@ -8,9 +8,6 @@ protocol AnyEndpoint: CustomStringConvertible {
     /// An identifier which uniquely identifies this endpoint (via its handler)
     /// across multiple compilations and executions of the web service.
     var identifier: AnyHandlerIdentifier { get }
-    
-    /// Description of the `Handler` this endpoint was generated for
-    var description: String { get }
```

Comment:
why is this removed here?

Reply:
`AnyEndpoint.description` is required by the `CustomStringConvertible` inheritance. I don't see why we would need to repeat that.
If `description` is supposed to have a different meaning, then I would remove the inheritance to avoid differing expectations.

Reply:
yep...totally right, thanks for clarifying!